schema.sql

```diff
@@ -65,3 +64,25 @@ CREATE TABLE visits (
     CONSTRAINT unique_visit_animal_vet UNIQUE (animal_id, vet_id, visit_date)
 );
 
+ALTER TABLE visits ADD COLUMN date_of_visit timestamp;
+
+SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+CREATE INDEX idx_animal_id ON visits(animal_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+CREATE INDEX idx_vet_id ON visits(vet_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE vet_id = 4;
+
+
+CREATE INDEX idx_email ON owners(email);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE email = 4;
```

Comment:
- [ ] You did great creating the indexes inside the `schema.sql` file. but the `EXPLAIN ANALYZE` are supposed to be inside the `querries.sql` file. I'll suggest you move all the three queeires `EXPLAIN ANALYZE SELECT COUNT(*)  FROM visits WHERE animal_id = 4;`, `EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE vet_id = 2;`, `SELECT * FROM owners where email = 'owner_18327@mail.com';` there. :smile: :+1: 

- [ ] Also your last 2 queries are different from the one that was in the requirement. this is what you have currently:
- - 
![Selection_006](https://github.com/Simpleshaikh1/vetclinic-db/assets/107632780/e74b208d-2be4-461e-badf-825a754ea81d)
 And this is what is on the requirement:
- -
![Selection_007](https://github.com/Simpleshaikh1/vetclinic-db/assets/107632780/79428403-c2d7-45dc-8a49-2b631268d19e)